packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -815,11 +834,39 @@ export function BudgetTable(props) {
   let currentMonth = monthUtils.currentMonth();
   let format = useFormat();
 
+  const mobileShowBudgetedColPref = useSelector(state => {
+    return (
+      (state.prefs.local && state.prefs.local.toggleMobileDisplayPref) || true
+    );
+  });
+
+  let [showBudgetedCol, setShowBudgetedCol] = useState(
+    !mobileShowBudgetedColPref &&
+      !document.cookie.match(/mobileShowBudgetedColPref=true/),
+  );
+
+  let dispatch = useDispatch();
+
+  function toggleDisplay() {
+    setShowBudgetedCol(!showBudgetedCol);
+    if (!showBudgetedCol) {
+      // remember the pref indefinitely
+      dispatch(savePrefs({ mobileShowBudgetedColPref: true }));
+    }
+  }
+  let buttonStyle = {
+    padding: 0,
+    backgroundColor: 'transparent',
```

Comment:
The corner triangle gets a little cut off with the button radius.
```suggestion
    backgroundColor: 'transparent',
    borderRadius: 'unset',
```